packages/nextjs/src/server/withClerkMiddleware.ts

```diff
@@ -1,5 +1,6 @@
 import type { AuthStatus, RequestState } from '@clerk/backend';
 import { constants, debugRequestState } from '@clerk/backend';
+import { AuthErrorReason } from '@clerk/backend/src/tokens/authStatus';
```

Comment:
🔧 i think we should not use non package-level exports since this couple more those 2 packages. If we want to use this i would suggest to either export it from the package or add a property to `requestState` like `isInterstitial` to be used instead of this check (eg `requestState.shouldSyncWithPrimary`).

Reply:
I will use the equivalent string representation for now